libs/pages/services/src/lib/feature/page-deployments-feature/page-deployments-feature.tsx

```diff
@@ -60,9 +60,18 @@ export function PageDeploymentsFeature() {
 
   useEffect(() => {
     const fetchEnv = () => {
-      dispatch(fetchEnvironmentDeploymentHistory({ environmentId }))
+      dispatch(fetchEnvironmentDeploymentHistory({ environmentId, silently: false }))
```

Comment:
Hello @WilliamTraoreee it would be very nice that silently is by default to false so we don't have to precise him in the normal case. We should have to precise the silently only when we want to use it. Can you rearrange a little bit the thing to get this behavior and then to remove the place where you set it to false?

Reply:
Thanks for your review ! I just made the param `silently` optional because I can't init it with a value but it works perfectly :D